PlaidLink.js

```diff
@@ -198,8 +186,9 @@ PlaidLink.propTypes = {
     }
   },
 
-  // An oauthStateId is required to support OAuth authentication flows when
-  // re-launching Link on iOS and using one or more European country codes.
+  // An oauthStateId is required to support OAuth authentication and payment flows when
+  // re-launching Link within a WebView and using one or more European country
+  // codes.
```

Comment:
These 3 oauth fields are only applicable to iOS, I guess it's fine to specify them for android, but we'll ignore them.